onnx/onnx.in.proto

```diff
@@ -95,13 +95,16 @@ enum Version {
   // - Implicitly add inference graph into each TrainingInfoProto's algorithm.
   IR_VERSION_2020_5_8 = 0x0000000000000007;
 
-  // IR VERSION 8 published on <TBD>
+  // IR VERSION 8 published on April 23, 2021
```

Comment:
2022

Reply:
I believe this is a reference to a previous version (an old TBD that was not updated earlier). But I am not sure what the date should be, April 23, 2021 was not the release date, I guess it was presumably the release date after that?

Reply:
I think it should be July 30, 2021. This date is the release date of ONNX 1.10 (the ONNX version which bumped IR version to 8).

Reply:
That makes more sense!